spec/services/catalog/update_order_item_spec.rb

```diff
@@ -39,7 +39,7 @@
         let(:service_instance) { TopologicalInventoryApiClient::ServiceInstance.new(:external_url => "external url") }
 
         before do
-          stub_request(:get, "http://localhost:3000/api/topological-inventory/v0.1/tasks/123")
+          stub_request(:get, "http://localhost:3000/api/topological-inventory/v1.0/tasks/123")
```

Comment:
We should be should be namespacing our specs, so we no longer need to specify the version here each time it changes in the `Gemfile`.

Is there a way the `v1.0` value can be shared? e.g. from the requestHelper?

Reply:
@syncrou I have redone the spec so its consistent with other specs that use the topology client but stub out the methods instead of stubbing out HTTP GET requests.